tensorflow_datasets/structured/movielens.py

```diff
@@ -48,22 +48,49 @@
 """
 
 _DESCRIPTION = """
-This dataset describes 5-star rating from MovieLens,
-a movie recommendation service.
-Users were selected at random for inclusion.
-There are 5 versions included: 25m, latest-small, 100k, 1m, 20m.
-In all datasets, the movies data and ratings data are joined on "movieId".
-The 25m dataset, latest-small dataset, and 1m dataset have the same schema
-and they do not include demographic data.
-The 25m dataset is the latest stable version of the MovieLens dataset.
-It is recommended for research purposes.
-The latest-small dataset is a small dataset with
-the same schema as 25m and 20m.
-It is updated over time by GroupLens and its checksum might need updating.
-The 20m dataset is one of the most used MovieLens datasets in academic papers
+This dataset contains a set of movie ratings from the MovieLens website, a movie
+recommendation service. This dataset was collected and maintained by [GroupLens]
+(https://grouplens.org/), a research group at the University of Minnesota. There
+are 5 versions included: "25m", "latest-small", "100k", "1m", "20m". In all
+datasets, the movies data and ratings data are joined on "movieId". The 25m
+dataset, latest-small dataset, and 20m dataset contain only movie data and
+rating data. The 1m dataset and 100k dataset contain demographic data in
+addition to movie and rating data.
+
+- "25m": This is the latest stable version of the MovieLens dataset. It is
+recommended for research purposes.
+- "latest-small": This is a small subset of the latest version of the MovieLens
+dataset. It is updated over time by GroupLens and its checksum might need
+updating.
+- "100k": This is the oldest version of the MovieLens datasets. It is a small
+dataset with demographic data.
+- "1m": This is the largest MovieLens dataset that contains demographic data.
+- "20m": This is one of the most used MovieLens datasets in academic papers
 along with the 1m dataset.
-The 100k dataset and 1m dataset are the oldest MovieLens datasets.
-They both contain demographic data and have the same schema.
+
+Feature Description:
```

Comment:
Perhaps remove the `Feature Description:` line?

Reply:
Removed in dfe803aa3e5f9ddba2d9e26214c612f095ea1b2a.